### PR TITLE
Change labels in sample collection form

### PR DESCRIPTION
**Pull request by allan-allay**
This closes: 
- https://github.com/onaio/rdt-standard/issues/324 & 
- https://github.com/onaio/rdt-standard/issues/326
See these referenced issues for details.

### CODE REVIEW

File: rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties [hunk 1]
```diff
@@ -20,12 +20,12 @@ sample-collection-form-original.step2.health_facility_type.v_required.err = Bagi
 sample-collection-form-original.step5.qr_code_reader.v_numeric.err =
 sample-collection-form-original.step1.next_label = TERUS
 sample-collection-form-original.step4.title = Sampel
-sample-collection-form-original.step2.name_of_primary_care_facility.options.other.text = Lainnya
+sample-collection-form-original.step2.name_of_health_facility.options.other.text = Lainnya
```
Comment (allan-allay) [inline rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties:23]: @vincent-karuri I noticed these are properties for a different step in the `sample-collection-form_in.properties` file. 
I was thinking maybe I shouldn't change these?

File: rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties [hunk 2]
```diff
@@ -63,19 +63,19 @@ sample-collection-form-original.step2.other_sample_type_lbl.text = Jenis Sampel
...
+sample-collection-form-original.step2.name_of_health_facility.options.dinas_kesehatan.text = Dinas Kesehatan
 sample-collection-form-original.step2.patient_health_status.label = Status kesehatan pasien?
 sample-collection-form-original.step2.sample_type.v_required.err = Bagian ini diperlukan
 sample-collection-form-original.step2.title = Koleksi sampel
 sample-collection-form-original.step3.next_label = TERUS
 sample-collection-form-original.step2.gsi_sample_number_lbl.text = Nomor Spesimen GSI
 sample-collection-form-original.step2.health_facility_type.options.clinic.text = Klinik
 sample-collection-form-original.step4.lbl_scan_respiratory_specimen_barcode.text = Pindai kode batang sampel
-sample-collection-form-original.step2.name_of_primary_care_facility.label = Nama Fasyankes
+sample-collection-form-original.step2.name_of_healthfacility.label = Nama Fasyankes
```
Comment (vincent-karuri) [inline rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties:78]: `name_of_healthfacility` looks wrong?

File: rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties (hunk 1 above)
Comment (vincent-karuri) [reply rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties:23]: The property file should be generated using `JMAG` acting on the `*-original.json`. The keys changed a bit after a form refactor, but these should reflect as new keys in Transifex - so not a problem.

Comment (vincent-karuri) [reply rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties:23]: `JMAG` should also be used to generate the `*.json` files from the `*-original.json` files.

Comment (vincent-karuri) [reply rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties:23]: So the flow is -> modify the `*-original.json`, generate 2 asset files against it using `JMAG` -> commit all three files. 

A caveat here is that for the placeholder-injected form generated by `JMAG`, remove the `-original` suffix in the `properties_file_name` field before commiting.

Comment (allan-allay) [reply rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties:23]: Cool. Thanks. Got the JMAG to generate the asset files after registering the Covid flavor widgets

File: rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties (hunk 2 above)
Comment (allan-allay) [reply rdt-open-reader/src/covidIndonesia/resources/sample-collection-form_in.properties]: Yeah, I've done this so that it's handled by `Transifex`

**Pull request merged**